src/ansys/fluent/core/workflow.py

```diff
@@ -991,18 +1164,36 @@ def __getattr__(self, attr):
         Parameters
         ----------
         attr : str
-            An attribute not defined in WorkflowWrapper
+            Attribute not defined in the ``WorkflowWrapper`` class.
```

Comment:
This class is `EnhancedWorkflowWrapper`, but I think that this documentation is highly redundant. Remove if permitted.